hcert_spec.md

```diff
@@ -147,19 +147,16 @@ The optical code is RECOMMENDED to be rendered on the presentation media with a
 
 Each Participating country is REQUIRED to provide a list of one or more Certificate Signing Certificate Authorities (CSCAs) and a list of all valid Document Signing Certificates (DSCs), and keep these lists current.
 
-The CSCA and DSC list format MUST follow the format of the ICAO master list (ldif) with the public key pairs packaged into an X.509v3 certificate as a base64 encoder DER.
-
-Each certificate:
+For the list of CSCA certificates, each certificate:
 
 - MUST contain A valid ‘C’ that matches the country of issuance.
-- MUST contain a well managed, unique, DN and unique Serial number
+- MUST contain a well managed, unique DN and unique Serial number
 - MUST contain a 256-bit Authority (Issuer) key identifier
 - MUST contain a 256-bit Subject key identifier
 
-In addition - each DSC certificate:
+In addition, for the list of DSC certificates, each certificate:
 
 - MUST contain validity range that is in line or broader than the EHC Validity Time of all EHC periods signed by that key.
-- SHOULD contain a X.509v3 Private Key Usage Period.
 - MUST contain a 256-bit Authority (Issuer) key identifier
 - MUST contain a 256-bit Subject key identifier
```

Comment:
There is no reason to place form requirements on an AKI. It simply MUST match the SKI of the signing cert, no matter how poor it is constructed and in what form or shape it has. The only actual requirement is that it is unique.

I also question that it MUST be present at all if the DSC is self signed.

In RFC 5280 most of this is regulated (https://tools.ietf.org/html/rfc5280#section-4.2.1.2). RFC 5280 says that SKI SHOULD be present. I personally think it is not needed as we will use the kid to identify keys and then will test them in order to determine suitability. So I would stick with RFC 5280 SHOULD, or simply remove that requirement.

Reply:
Just updated my comment above.

Reply:
Can you suggest an edit?

Reply:
Providing a suggested edit below. Some comments to this edit:

- The original requirement on a unique serial number in a CSCA is removed since a serial number is only unique within the batch of certificates issued by a particular CA. Since the CSCA in most cases is self signed, the uniqueness requirement is relevant. All CSCA certs could have serial number = 1.
- Choosing to refer to RFC 5280 for requirements on SKI. The bottom line is that there is only a requirement for uniqueness. Perhaps RFC 5280 should be referred to as a general requirement, but I didn't do that for one particular reason and that is that RFC 5280 only applies to certificates issued by a CA. It does not contain self issued certificates and both CSCA as well as DSC certificates can be self signed.

**Proposed edit:**

For the list of CSCA certificates, each certificate:

- MUST contain A valid ‘C’ that matches the country of issuance.
- MUST contain DN that is unique within the specified country.
- MUST contain a unique Subject key identifier according to ([RFC5280](https://tools.ietf.org/html/rfc5280))

In addition, for the list of DSC certificates, each certificate:

- MUST contain validity range that is in line or broader than the EHC Validity Time of all EHC periods signed by that key.
- MUST, if the DSCA certificate is signed by a CSCA certificate, contain an Authority key identifier matching the Subject key identifier of the issuing CSCA certificate.
- SHOULD contain a unique Subject key identifier

Reply:
Made some final edits to the text above

Reply:
I've updated the text accordingly